multi-channel-chat/http-channel/src/lib.rs

```diff
@@ -0,0 +1,102 @@
+use chatlog::*;
+use outbound::*;
+use serde::{Deserialize, Serialize};
+use wasmbus_rpc::actor::prelude::*;
+use wasmcloud_interface_httpserver::*;
+use wasmcloud_interface_logging::debug;
+use wasmcloud_interface_numbergen::*;
+
+#[allow(dead_code)]
+mod chatlog;
+
+#[allow(dead_code)]
+mod outbound;
+
+const CHANNEL_NAME: &str = "http";
+const API_ACTOR: &str = "mcchat/api";
+
+#[derive(Debug, Default, Actor, HealthResponder)]
+#[services(Actor, HttpServer, Outbound)]
+struct HttpChannelActor {}
+
+#[async_trait]
+impl Outbound for HttpChannelActor {
+    async fn publish_message(&self, ctx: &Context, arg: &OutboundMessage) -> RpcResult<bool> {
+        // This is absorbed silently because the HTTP channel does not currently expose
+        // any kind of realtime subscription. Perhaps in the future a websocket subscription
+        // could be used?
+        Ok(true)
+    }
+}
+
+/// Implementation of HttpServer trait methods
+#[async_trait]
+impl HttpServer for HttpChannelActor {
+    async fn handle_request(&self, ctx: &Context, req: &HttpRequest) -> RpcResult<HttpResponse> {
+        let path = &req.path[1..req.path.len()];
```

Comment:
Nit: Might just want to use `trim_start_matches` below instead of slicing

Reply:
counter nit: this is the code from all of our HTTP server examples